tests/locust/tests/capacity_test.py

```diff
@@ -0,0 +1,154 @@
+import csv
+import os
+import time
+import string
+import uuid
+from locust import HttpUser

```

Comment:
can import multiple things from `locust` at the same time:

`from locust import HttpUser, TaskSet, task, constant, LoadTestShape, between`